ingest/src/main/java/com/microsoft/azure/kusto/ingest/MappingConst.java

```diff
@@ -0,0 +1,25 @@
+package com.microsoft.azure.kusto.ingest;
+
+public enum MappingConst {
+    // Json Mapping consts
+    PATH("Path"),
```

Comment:
Like I said above, you could just expose these as methods on the class

Reply:
please see if your comment is still relevant after last code iteration